spirv_hlsl.cpp

```diff
@@ -567,250 +567,329 @@ void CompilerHLSL::emit_builtin_outputs_in_struct()
 	auto &execution = get_entry_point();
 
 	bool legacy = hlsl_options.shader_model <= 30;
-	active_output_builtins.for_each_bit([&](uint32_t i) {
```

Comment:
Is this just a formatting change? This is un-reviewable, the file has 2k line additions.